libs/spark/src/Unstable_SectionMessage/Unstable_SectionMessage.stories.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { Unstable_SectionMessage } from '..';
+
+export default {
+  title: '@ps/Unstable_SectionMessage',
+  component: Unstable_SectionMessage,
+} as Meta;
+
+const Template = (args) => (
+  <Unstable_SectionMessage {...args}>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+  </Unstable_SectionMessage>
+);
+
+export const Configurable: Story = Template.bind({});
+
+export const Default: Story = Template.bind({});
+Default.storyName = '(default)';
+
+export const SeverityInfo: Story = Template.bind({});
+SeverityInfo.args = { severity: 'info' };
+SeverityInfo.storyName = 'severity=info';
+
+export const SeverityInfoOnClose: Story = Template.bind({});
+SeverityInfoOnClose.args = {
+  severity: 'info',
+  onClose: () => {
+    return;
+  },
+};
+SeverityInfoOnClose.storyName = 'severity=info onClose';
```

Comment:
```suggestion
const onClose = () => { return; };

export const SeverityInfoOnClose: Story = Template.bind({});
SeverityInfoOnClose.args = { severity: 'info', onClose };
SeverityInfoOnClose.storyName = 'severity=info onClose';
```

It's arbitrary, but we can save some space if you extract this arg that takes 3 lines every time its set. Carry through to below stories as well.